projects/angular-auth-oidc-client/src/lib/auto-login/auto-login-all-routes.guard.ts

```diff
@@ -8,7 +8,7 @@ import { LoginService } from '../login/login.service';
 import { AutoLoginService } from './auto-login.service';
 
 @Injectable({ providedIn: 'root' })
-export class AutoLoginAllRoutesGuard implements CanActivate, CanActivateChild, CanLoad {
+export class AutoLoginAllRoutesGuard  {
```

Comment:
TODO: revert this (this was automatically done by the `ng update` command because class guards are deprecated in favor of functional guards).
Maybe that we should deprecate these classes?